CODE_OF_CONDUCT.md

```diff
@@ -33,16 +33,18 @@ Examples of unacceptable behavior by participants include:
 
 ## Our Responsibilities
 
-Project maintainers are responsible for clarifying the standards of acceptable
+Committers, Managers, and Administrators are responsible for clarifying the standards of acceptable
 behavior and are expected to take appropriate and fair corrective action in
 response to any instances of unacceptable behavior.
 
-Project maintainers have the right and responsibility to remove, edit, or
+Project Managers and Administrators have the right and responsibility to remove, edit, or
 reject comments, commits, code, wiki edits, issues, and other contributions
 that are not aligned to this Code of Conduct, or to ban temporarily or
-permanently any contributor for other behaviors that they deem inappropriate,
+permanently any Contributor for other behaviors that they deem inappropriate,
 threatening, offensive, or harmful.
 
+For more information on the various roles and responsibilities, [visit our wiki](https://vufind.org/wiki/community:roles_and_responsibilities).
```

Comment:
We should also probably link to the governance document here after #1890 is merged. I'll add a TODO.